### DISCUSSION TITLE
Welcome to Mouwrice Discussions!

### DISCUSSION THREAD

**Discussion by Mouwrice** [Mouwrice/Mouwrice#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Mouwrice** [comment]
what it do?

**Reply by Nyxes5** [reply]
it discusses

**Comment by Mouwrice** [comment]
cool cool cool

**Reply by lars-vc** [reply]
P O G

**Comment by lars-vc** [comment]
***comment with style***

**Comment by Mouwrice** [comment]
how i close?

**Reply by stijndcl** [reply]
You don't close the discussion, the discussion closes you

**Comment by Mouwrice** [comment]
![image](https://user-images.githubusercontent.com/56763273/130800506-afafc82f-9f6c-4db3-a936-47babe39c2ac.png)
